awaitwhat/dot.py

```diff
@@ -47,6 +49,22 @@ def describe(task, current=None):
     return f"[label={label}]"
 
 
+def graph(tasks, current=None) -> Tuple[Set[Task], Set[Tuple[Task, Task]]]:
```

Comment:
`current` is not used, is that intentional?

Reply:
Thanks, done!